src/components/activity-page/managed-interactive/managed-interactive.tsx

```diff
@@ -59,7 +59,7 @@ export const ManagedInteractive: React.ForwardRefExoticComponent<IProps> = forwa
     }
   }, [embeddableRefId, shouldWatchAnswer]);
 
-    const handleNewInteractiveState = (state: IRuntimeMetadata) => {
+    const handleNewInteractiveState = (state: any) => {
```

Comment:
I don't think we should be dropping down to `any` here is we can help it.

Reply:
This any is one of the key parts of this PR. Interactives can send us pretty much anything. objects, strings, numbers, booleans.

Reply:
I've handled this in other repos by aliasing the `any` type (eg `type IRuntimeState = any`).  That allows you to trace a typed value through the code whereas directly using `any` doesn't.

Reply:
I wasn't going to try to add some type checking here:
https://github.com/concord-consortium/activity-player/pull/145/files#diff-51cdda74ca9043dfc36956d3c97861042a6434794e12e2b8f80f6d760a3d9e16R92-R99

But I wasn't sure yet how to do it. At least that way someone editing the function can't try to reference a property that isn't defined by the interactive api.

Reply:
I just checked and aliases don't seem to provide any benefit more than documentation. 
If I have a function with an argument of InteractiveState and InteractiveState is aliased to any, then I can pass any type to this function. In other words it won't force me to cast to InteractiveState. That makes some sense when I think about it.

So adding this type doesn't seem worth it since it doesn't guarantee the documentation will be up-to-date. Maybe for an exposed API of a library it is useful, but in internal code I don't see the benefit.

Reply:
Right - `any` matches anything for the compiler but when you are in an IDE you can right click on the type and find all references to it - that's what I meant by "trace a typed value though the code".  Sorry I wasn't clear.

Reply:
>I was going to try to add some type checking ... But I wasn't sure yet how to do it

I mentioned this in another comment elsewhere, but [type guards](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards) can be useful for doing run-time type-checking in a way that feeds into the type system.

Reply:
I need to get this out the door, so I didn't do anything more with this `any` above.
If I had more time I'd at least switch it to `unknown` since that is more accurate than `any`.

I don't see the benefit of the alias since it doesn't guarantee searches for that type will find all of the instances since a developer could decide to use any (or unknown) or its alias and typescript wouldn't complain.

Reply:
I did add more type checking to `getAnswerWithMetadata` (referenced in the code URL above)